nixos/modules/services/cluster/dynomite.nix

```diff
@@ -0,0 +1,60 @@
+{ config, pkgs, lib, ... }: with lib; let
+  cfg = config.services.dynomite;
+  configFile = pkgs.writeText "dynomite.yaml" "${cfg.config}";
```

Comment:
You might be interested in https://github.com/NixOS/rfcs/pull/42.

Reply:
One of the benefits here is that you could automatically connect to the local `redis` server and include the port.

`services.dynomite.settings.servers = [ "127.0.0.1:${config.services.redis.port}" ];`

Reply:
Is there any sensitive data that needs to be stored in this file?

Reply:
the tokens are pre shared secrets, should i move it some where else?